src/software/ai/navigator/navigator.h

```diff
@@ -151,8 +158,9 @@ class Navigator : public IntentVisitor
     // This variable is set by each `visit` function
     std::unique_ptr<Primitive> current_primitive;
 
-    // This is used by the visualizer to see the planned paths
+    // The following are exposed to users of the Navigator
```

Comment:
this is a super nit, but the exposing of these members is already "obvious" by the public interface of the class. I think this comment is a bit redundant and at risk of not being updated should something change, so think we'd be ok without it.

Reply:
removed it